### PR TITLE
Add version: ShiningLight.OpenSSL version 1.1.1v

### PR DESCRIPTION
**Pull request by szhu25**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/114622&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/s/ShiningLight/OpenSSL/1.1.1v/ShiningLight.OpenSSL.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - DisplayName: OpenSSL 3.0.8 (64-bit)
+    Publisher: OpenSSL Win64 Installer Team
+    ProductCode: OpenSSL (64-bit)_is1
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://slproweb.com/download/Win32OpenSSL-1_1_1v.exe
+  InstallerSha256: DF0282FBC54374A8B7A5F8090279A89105EFBA2017724D03E3A00BCB2FE890AA
+  ProductCode: OpenSSL (32-bit)_is1
+  AppsAndFeaturesEntries:
+  - DisplayName: OpenSSL 3.0.8 (32-bit)
```
Comment (ItzLevvie) [inline manifests/s/ShiningLight/OpenSSL/1.1.1v/ShiningLight.OpenSSL.installer.yaml:45]:
```suggestion
  - DisplayName: OpenSSL 1.1.1v (32-bit)
```

File: manifests/s/ShiningLight/OpenSSL/1.1.1v/ShiningLight.OpenSSL.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  InstallerUrl: https://slproweb.com/download/Win32OpenSSL-1_1_1v.msi
+  InstallerSha256: EAFC6B87D6FA6DFB31E7F57CB96975A6FCDD4FA5F1F4B0C4F847C0C69B494B8E
+  ProductCode: '{3FE104A2-C100-4E96-A1E9-5C18EEAD1693}'
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://slproweb.com/download/Win64OpenSSL-1_1_1v.exe
+  InstallerSha256: A952A173AECCF858F99F8D9D5740B135CD2B425289F60D899BE2FC9E462AA9B3
+  ProductCode: OpenSSL (64-bit)_is1
+  AppsAndFeaturesEntries:
+  - DisplayName: OpenSSL 3.0.8 (64-bit)
```
Comment (ItzLevvie) [inline manifests/s/ShiningLight/OpenSSL/1.1.1v/ShiningLight.OpenSSL.installer.yaml:36]:
```suggestion
  - DisplayName: OpenSSL 1.1.1v (64-bit)
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Hi @szhu25,

This package appears to require a dependency:
image

The `x86` version requires the `x86` dependency.

**Comment by szhu25** [comment]
> Hi @szhu25,
> 
> This package appears to require a dependency:
> image
> 
> The `x86` version requires the `x86` dependency. 

Hmm that makes sense... 
I'll retest all 6 variants of the OpenSSL installer in the sandbox when I got home tonight. 
Although I thought OpenSSL have this dependency requirement for a while now, maybe they recently modified the C++ library? 

Would this still be able to submit if this dependency requirement exists?

**Comment by szhu25** [comment]
Happy Weekends @stephengillie,

I tested this manifest using Windows sandbox (via the sandbox script in this repo) and this software installed without any issues... Is it possible to restart the pipeline and check?

The developer of this software build 6 versions of OpenSSL (OpenSSL 1.1.1v, 3.0.10, 3.1.2 and its corresponding light version), so far I have not seen the no dependency error on any of my Windows Sandbox test... I do admit that I have 2015 - 2022 installed on my host machine, but I don't know if it would affect Sandbox env?

**Comment by stephengillie** [comment]
The error is still present when installed in a Windows 10 VM: 
image

It's from the Inno installers.

**Pull request closed without merging**